cip/1.accepted/CIP2017-02-06-Regular-Path-Patterns.adoc

```diff
@@ -0,0 +1,160 @@
+= CIP2017-02-06 Regular Path Patterns
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Authors:* Tobias Lindaaker <tobias.lindaaker@neotechnology.com>
+
+toc::[]
+
+== Regular Path Patterns
+
+Above and beyond the types of patterns that can be expressed in Cypher using the normal path syntax, Cypher also supports what amounts to regular expressions over paths.
+This functionality is called Regular Path Patterns.
+
+A Regular Path Pattern is defined as:
+
+• A simple relationship type, or
+• A Regular Path Pattern followed by another Regular Path Pattern, or
+• An alternative between two Regular Path Patterns, or
+• A repetition of a Regular Path Pattern, or
+• A reference to a Defined Path Predicate.
+
+Regular Path Patterns are written similarly to how relationship patterns are written, but enclosed within two slash (`/`) characters instead of brackets (`[]`).
+
+Contrary to Relationship Patterns, Regular Path Patterns do _not_ allow binding a relationship to a variable.
+In order to bind the matching path to a variable, a Path Assignment should be used, by preceding the path with an identifier and an equals sign (`=`).
+This avoids a problem that existed in the past with repetition of relationships (a syntax that was deprecated with the introduction of Regular Path Patterns), where a relationship variable would bind to a list, making it hard to express predicates over the actual relationships.
```

Comment:
I've tried writing the text as if Regular Path Patterns are already in the language, since when this text is merged, it will be in the language. Although _is_ works in that case as well.